### PR TITLE
Add a blog post to announce Debian Security Tracker integration

### PR DESCRIPTION
**Pull request by hogo6002**
Add a blog post to announce Debian Security Tracker integration

### CODE REVIEW

File: gcp/appengine/blog/content/posts/supporting-debian-security-tracker-data/index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Supporting Debian Security Tracker data"
+date: 2024-06-18T00:00:00Z
+draft: false
+author: Holly Gong
+---
+
+OSV.dev aims to be a comprehensive database for all known vulnerabilities in open source ecosystems. One of the more recent areas of focus is Linux distributions, which are important in the context of container scanning. To enable comprehensive scanning support for Debian based containers, we've integrated more vulnerability data from Debian Security Tracker into our OSV database.
```
Comment (andrewpollock) [inline gcp/appengine/blog/content/posts/supporting-debian-security-tracker-data/index.md:8]: Differentiate between scanning running containers versus container images from a registry

```suggestion
OSV.dev aims to be a comprehensive database for all known vulnerabilities in open source ecosystems. One of the more recent areas of focus is Linux distributions, which are important in the context of container image scanning. To enable comprehensive scanning support for Debian based containers images, we've integrated more vulnerability data from Debian Security Tracker into our OSV database.
```

File: gcp/appengine/blog/content/posts/supporting-debian-security-tracker-data/index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+- **Important vulnerabilities:** Vulnerabilities tagged as "low", "medium", "high", and "end-of-life" are added into our database into corresponding CVE entries. These vulnerabilities will be displayed when scanning a Debian image. 
+- **Unimportant vulnerabilities:** Vulnerabilities marked as "unimportant" are also added into our database but are hidden by default in OSV-Scanner results. This is because those vulnerabilities typically do not affect users, but in some rare cases, they might. In the future, when container scanning for Debian Docker images is fully implemented, users can choose whether to include these vulnerabilities in their reports.
+- **Untriaged vulnerabilities:** Vulnerabilities marked as "not yet assigned" are not added to our database, as the Debian team may soon evaluate and update their status.
+
+## Coming Soon: More comprehensive Container Scanner in OSV-Scanner
+OSV-Scanner has previously focused on being a vulnerability scanner that scans code repositories, with limited support for Docker containers. This year, we’re focusing on providing more full featured support for Docker container scanning.
```
Comment (andrewpollock) [inline gcp/appengine/blog/content/posts/supporting-debian-security-tracker-data/index.md:32]: Is there somewhere interested readers can follow along on this work, like a specific GitHub Issue?

File: gcp/appengine/blog/content/posts/supporting-debian-security-tracker-data/index.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+To maintain accuracy while expanding our database, we've made the following decisions on how to prioritize these vulnerabilities:
+
+- **Important vulnerabilities:** Vulnerabilities tagged as "low", "medium", "high", and "end-of-life" are added into our database into corresponding CVE entries. These vulnerabilities will be displayed when scanning a Debian image. 
+- **Unimportant vulnerabilities:** Vulnerabilities marked as "unimportant" are also added into our database but are hidden by default in OSV-Scanner results. This is because those vulnerabilities typically do not affect users, but in some rare cases, they might. In the future, when container scanning for Debian Docker images is fully implemented, users can choose whether to include these vulnerabilities in their reports.
+- **Untriaged vulnerabilities:** Vulnerabilities marked as "not yet assigned" are not added to our database, as the Debian team may soon evaluate and update their status.
+
+## Coming Soon: More comprehensive Container Scanner in OSV-Scanner
```
Comment (andrewpollock) [inline gcp/appengine/blog/content/posts/supporting-debian-security-tracker-data/index.md:31]:
```suggestion
## Coming Soon: More comprehensive Container Image Scanning in OSV-Scanner
```

File: gcp/appengine/blog/content/posts/supporting-debian-security-tracker-data/index.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+## Why do we need it?
+Previously, OSV relied solely on Debian Security Advisories (DSAs) and Debian Long Term Support Advisories (DLAs) for Debian related vulnerability information. While these are official sources, they only cover issues that have been formally [acknowledged and patched](https://www.debian.org/doc/manuals/securing-debian-manual/dsa.en.html) by the Debian team. This left a blind spot in our scanning reports – unfixed vulnerabilities, where the security issue has been reported but a fix has not been made available yet in Debian.
```
Comment (andrewpollock) [inline gcp/appengine/blog/content/posts/supporting-debian-security-tracker-data/index.md:18]: A little bit of a nit, Debian's a project, the security team largely handle security fixes, so either refer to it overall as Debian, or the Debian security team in this instance. Also differentiate between OSV and OSV.dev. Also link to more information about the current conversion process, for the curious.

```suggestion
Previously, OSV.dev [relied](https://google.github.io/osv.dev/data/#converted-data) solely on Debian Security Advisories (DSAs) and Debian Long Term Support Advisories (DLAs) for Debian related vulnerability information. While these are official sources, they only cover issues that have been formally [acknowledged and patched](https://www.debian.org/doc/manuals/securing-debian-manual/dsa.en.html) by Debian. This left a blind spot in our data coverage – unfixed vulnerabilities, where the security issue has been reported but a fix has not been made available yet in Debian.
```

**Review by andrewpollock — APPROVED** [review]
LGTM with a few nitty suggestions.

File: gcp/appengine/blog/content/posts/supporting-debian-security-tracker-data/index.md (hunk 2 above)
Comment (hogo6002) [reply gcp/appengine/blog/content/posts/supporting-debian-security-tracker-data/index.md:32]: @another-rex do we have any public docs or issues for container image scanning

File: gcp/appengine/blog/content/posts/supporting-debian-security-tracker-data/index.md (hunk 3 above)
Comment (hogo6002) [reply gcp/appengine/blog/content/posts/supporting-debian-security-tracker-data/index.md]: Thanks Andrew!

File: gcp/appengine/blog/content/posts/supporting-debian-security-tracker-data/index.md (hunk 2 above)
Comment (cuixq) [reply gcp/appengine/blog/content/posts/supporting-debian-security-tracker-data/index.md:32]: this one? https://github.com/google/osv-scanner/issues/64

**Pull request merged**